### PR TITLE
Add section on final Calculator Project

### PR DESCRIPTION
**Pull request by digitopolis**
- initial project files
- README with instructions/requirements

### CODE REVIEW

File: 6-calculator-project/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+- Before you start doing any calculations, first just make sure the display correctly updates when each number key is clicked. Since the display is an `<input>` element, you can use its `value` property, instead of `innerText` like you've done before. You could do something like this
+```js
+  const display = document.querySelector('.calculator-screen');
+  display.value = // set display value here
+```
+
+- There are a few different pieces of data that you'll need to be able to keep track of (and keep updated) as a user . You could handle this in various ways – one possible solution might be to create a `calculator` object to hold each piece of changing information. This would help keep everything in once place, and allow you to pass the whole object to functions if necessary. It might look something like this
```
Comment (k3ntako) [inline 6-calculator-project/README.md:21]: `as a user .` no need for space?

Comment (k3ntako) [reply 6-calculator-project/README.md:21]: maintaining state is going to be hard for them

**Review by k3ntako — COMMENTED** [review]
Left some comments but looks good! 

Is there a completed example they can see? You don't need to build it if they didn't provide one!

**Review by k3ntako — APPROVED** [review]

File: 6-calculator-project/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - subtraction `-`
+  - multiplication `×` (represented in HTML with `&times;`)
+  - division `÷` (represented in HTML with `&divide;`)
+- Clicking each number button should add a digit to the number in the display
+- Clicking the AC button should clear the display back to "0" (zero)
+  - It also deselects any arithmetic operators
+- Selecting an operator updates the display with the correct symbol
+- After an operator is selected the display should clear to be ready to start displaying the second operand
+- Clicking the equals `=` button should update the display with the correct result of the operation
+  - It should also reset the operator dropdown so no operator is selected
```
[the remark this answers is not in the document]
Comment (digitopolis) [reply 6-calculator-project/README.md:47]: I like that! A little more complex for a later feature to tackle maybe?

File: 6-calculator-project/README.md (hunk 1 above)
Comment (digitopolis) [reply 6-calculator-project/README.md:21]: `as a user .` <-- I didn't finish the sentence 😅 Thanks for catching

State will be a challenge.. for sure

**Pull request merged**